monitoring/alertmanager/config.yml

```diff
@@ -0,0 +1,11 @@
+global:
+  slack_api_url: 'https://hooks.slack.com/services/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'
```

Comment:
This is a secret info and should be removed from the git repo. Place this in gitlab CI/CD variables and use a docker entrypoint to template your alertmanager config